### PR TITLE
add kube-logging-operator (latest)

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

**Review by imjasonh — DISMISSED** [review]

File: kube-logging-logging-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          # The helm chart expects the logging-operator binaries to be in / instead of /usr/bin
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+      - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: kube-logging/logging-operator
+    tag-filter: 4.2.2
```
Comment (imjasonh) [inline kube-logging-logging-operator.yaml:44]: This won't match any other tags so this won't update automatically

File: kube-logging-logging-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package:
+  name: kube-logging-logging-operator
```
Comment (rawlingsj) [inline kube-logging-logging-operator.yaml:2]: looks like there's an extra `-logging` in the name here

Comment (developer-guy) [reply kube-logging-logging-operator.yaml]: yep this was on purpose but I updated, thanks.

File: kube-logging-logging-operator.yaml (hunk 1 above)
Comment (developer-guy) [reply kube-logging-logging-operator.yaml]: I fixed it, thank you.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**